server.go

```diff
@@ -601,41 +606,86 @@ type nodeAddresses struct {
 // sendToPeer send a message to the server telling it to send the specific set
 // of message to a particular peer. If the peer connect be found, then this
 // method will return a non-nil error.
-func (s *server) sendToPeer(target *btcec.PublicKey, msgs ...lnwire.Message) error {
-	errChan := make(chan error, 1)
-
+// handleSend attempts to forward a sendReq to the outgoing queue of a
+// particular peer. Access to the peers map is synchronized to prevent changes
+// in set membership during lookup. The process of queueing the message and
+// signaling back to the caller is done asynchronously to free up the
+// queryHandler.
+func (s *server) SendToPeer(target *btcec.PublicKey, msgs ...lnwire.Message) error {
 	msgsToSend := make([]lnwire.Message, 0, len(msgs))
 	msgsToSend = append(msgsToSend, msgs...)
-	sMsg := &sendReq{
-		target:  target,
-		msgs:    msgsToSend,
-		errChan: errChan,
+
+	s.mu.Lock()
+	defer s.mu.Unlock()
+
+	return s.sendToPeer(target, msgsToSend)
+}
+
+func (s *server) sendToPeer(target *btcec.PublicKey, msgs []lnwire.Message) error {
+	// Compute the target peer's identifier.
+	targetPubBytes := target.SerializeCompressed()
+
+	srvrLog.Infof("Attempting to send msgs %v to: %x",
+		len(msgs), targetPubBytes)
+
+	// Lookup intended target in peersByPub, returning an error to the caller if
+	// the peer is unknown.  Access to peersByPub is synchronized here to ensure
```

Comment:
This line and a few others are over the 80-character limit observed uniformly throughout the codebase. I think you might need modify how your editor interprets tabs?